baekjoon/stick.js

```diff
@@ -0,0 +1,14 @@
+const input = require('fs').readFileSync('/dev/stdin').toString().split('\n').slice(1, -1).map((value) => Number(value));
+
+let rightNum = input.pop();
+let cnt = 1;
+
+while (input.length) {
+    const compareNum = input.pop();
+    if (compareNum > rightNum) {
+        cnt ++;
```

Comment:
공백 제거[(space-unary-ops)](https://eslint.org/docs/rules/space-unary-ops)
```suggestion
        cnt++;
```